core/ts/Poll.ts

```diff
@@ -655,6 +655,11 @@ export class Poll implements IPoll {
 
     // ensure we pass the dynamic tree depth
     circuitInputs.actualStateTreeDepth = this.actualStateTreeDepth.toString();
+    const coordinatorPublicKey = this.coordinatorKeypair.pubKey;
```

Comment:
I think you can directly use the `PubKey` class method `hash` https://github.com/privacy-scaling-explorations/maci/blob/dev/domainobjs/ts/publicKey.ts#L86